hw5/code/hw6_script.py

```diff
@@ -0,0 +1,120 @@
+###############################################################################################
+# this script is a copy of a notebook with the 6th homework for pandas and matplotlib/seaborn #
+###############################################################################################
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+
+## the first part: pandas
+
+# let's read *.ggf
+def read_gff(path):
+    return pd.read_csv(path, sep="\t", comment="#",
+                       names=["chromosome", "source", "type", "start", "end",
+                              "score", "strand", "phase", "attributes"])
+
+
+gff = read_gff('rrna_annotation.gff')
```

Comment:
Очень круто, что ты залила данные в репозиторий и ещё выделила их в отдельные папки, это очень хорошая практика. Но в идеале код должен работать "из коробки". Склонировал репозиторий -> открыл ноутбук -> выполнил код, никаких лишних действий. Тут же указан не существующий путь до данных
```suggestion
gff = read_gff('../data/rrna_annotation.gff')
```